src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -172,7 +172,7 @@ public static void serializeNetworkData(@NotNull final Colony colony, @NotNull f
         BlockPosUtil.writeToByteBuf(buf, colony.getCenter());
         buf.writeBoolean(colony.isManualHiring());
         //  Citizenry
-        buf.writeInt(colony.getCitizenManager().getCurrentCitizenCount());
+        buf.writeInt(colony.getCitizenManager().getMaxCitizens());
 
```

Comment:
What is wrong with the current citizen count?

Reply:
Well, nothing is wrong with it. It's just when this is read in, it's read into a variable that is supposed to represent the maximum citizen count. So for example it would show current/current in the gui instead of current/max.